src/GeoExt/data/proxy/Protocol.js

```diff
@@ -22,27 +22,67 @@ Ext.define('GeoExt.data.proxy.Protocol', {
     ],
     alias: 'proxy.gx_protocol',
 
-    /**
-     * The protocol used to fetch features.
-     *
-     * @cfg {OpenLayers.Protocol}
-     */
-    protocol: null,
+    config:  {
 
-    /**
-     * Abort any previous request before issuing another.
-     *
-     * @cfg {Boolean}
-     */
-    abortPrevious: true,
+        /**
+         * The protocol used to fetch features.
+         *
+         * @cfg {OpenLayers.Protocol}
+         */
+        protocol: null,
+
+        /**
+         * Abort any previous request before issuing another.
+         *
+         * @cfg {Boolean}
+         */
+        abortPrevious: true,
+
+        /**
+         * Should options.params be set directly on options before passing it
+         * into the protocol's read method?
+         *
+         * @cfg {Boolean}
+         */
+        setParamsAsOptions: false,
+
+        /**
+         * The reader to use.
+         *
+         * @cfg {Ext.data.reader.Reader}
+         */
+        reader: {}
+
+    },
 
     /**
-     * Should options.params be set directly on options before passing it into
-     * the protocol's read method?
+     * We need to override this as the OpenLayers classes passed as configs
+     * loose their class-nature and seem to be copied by ExtJS somewhere.
+     *
+     * We deal with this elsewhere in a different manor and should see if
+     * we can either share code or get rid of this special handling all
+     * together. The problem isn't reproducible for other 'classes' with a
+     * similar inheritance startegy as OpenLayers 2 has.
      *
-     * @cfg {Boolean}
+     * TODO Find a way to have this functionality shared or get rid of it.
      */
-    setParamsAsOptions: false,
+    initConfig: function(config){
+        var me = this,
+            cfg = config || {},
+            prefix = me.$configPrefixed ? '_' : '',
```

Comment:
I don't know if it is standard, I found out about it via reverse engineering. As this whole method is more or less a workaround and it works for both versions, I'd leave it in.

Before the extjs5 branch is ready for prime time, every change will be re-reviewed for sure.